pdns/rec_channel.cc

```diff
@@ -17,6 +17,8 @@
 
 #include "namespaces.hh"
 
+volatile sig_atomic_t RecursorControlChannel::stop;
```

Comment:
Is the initial value set somewhere?

Reply:
https://en.cppreference.com/w/cpp/language/initialization defines that statics are zero initialized. But it might be better to do it explicitly indeed .